### PR TITLE
build: update gobra to support the new rules of gofmt

### PR DESCRIPTION
**Pull request by jcp19**
@oncilla 
This PR updates the version of Gobra used in the CI to support the new rewriting rules for gofmt as of Go 1.19.  After merging this, PR https://github.com/scionproto/scion/pull/4271 should verify as is.

- - -
This change is [Reviewable](https://reviewable.io/reviews/scionproto/scion/4279)

### CODE REVIEW

**Comment by jcp19** [comment]
Currently, the CI is failing with errors that I did not observe locally. I will make this PR a draft while I'm debugging it

**Comment by jcp19** [comment]
@oncilla should be fixed now.

I had to introduce a new file (`util_spec.gobra`) because Gobra now requires that all imported packages are populated (i.e., have at least one annotated file). This is a recent change which has to do with the support for specifying and verifying initialization code. Maybe this check is a bit too strict, so we may drop it in the future. At any rate,  `util_spec.gobra` will be useful to verify the method `String()` in `infofield.go`, when Gobra supports the `fmt` package.

**Review by matzf — APPROVED** [review]
:lgtm:

> Reviewed 2 of 2 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/scionproto/scion/4279#-:-NEBtlc36qR3fadyhlxB:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @jcp19)

**Review by lukedirtwalker — CHANGES_REQUESTED** [review]
> Reviewed 1 of 2 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/scionproto/scion/4279)* status: all files reviewed, 1 unresolved discussion (waiting on @jcp19)

___
*[`pkg/private/util/util_spec.gobra` line 23 at r2](https://reviewable.io/reviews/scionproto/scion/4279#-NEEz7T1-qS7Mtass57V:-NEEz7T1-qS7Mtass57W:b-yhiri8) ([raw file](https://github.com/scionproto/scion/blob/bb004abd4bdc97d9806badac9ff5f2ae44faf7bf/pkg/private/util/util_spec.gobra#L23)):*
> ```
> 
> decreases
> func SecsToCompact(t uint32) string
> ```

please add a newline at the end of the file.

File: pkg/private/util/util_spec.gobra [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+// This file contains the specification of some methods from package
+// util which haven't been verified.
+
+// +gobra
+
+package util
+
+decreases
+func SecsToCompact(t uint32) string
```
Comment (jcp19) [inline pkg/private/util/util_spec.gobra]:
```suggestion
func SecsToCompact(t uint32) string

```

**Review by jcp19 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/scionproto/scion/4279)* status: 1 of 2 files reviewed, 2 unresolved discussions (waiting on @lukedirtwalker and @matzf)

___
*[`pkg/private/util/util_spec.gobra` line 23 at r2](https://reviewable.io/reviews/scionproto/scion/4279#-NEEz7T1-qS7Mtass57V:-NEF87LQDcicGO1YplOj:b-896fix) ([raw file](https://github.com/scionproto/scion/blob/bb004abd4bdc97d9806badac9ff5f2ae44faf7bf/pkg/private/util/util_spec.gobra#L23)):*
<details><summary><i>Previously, lukedirtwalker (Lukas Vogel) wrote…</i></summary><blockquote>

please add a newline at the end of the file.
</blockquote></details>

Done.

**Review by lukedirtwalker — APPROVED** [review]
> Reviewed 1 of 2 files at r2, 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/scionproto/scion/4279)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @jcp19)

**Pull request merged**